src/pip/_internal/__init__.py

```diff
@@ -4,50 +4,31 @@
 import locale
 import logging
 import os
-import warnings
-
 import sys
+import warnings
 
-# 2016-06-17 barry@debian.org: urllib3 1.14 added optional support for socks,
-# but if invoked (i.e. imported), it will issue a warning to stderr if socks
-# isn't available.  requests unconditionally imports urllib3's socks contrib
-# module, triggering this warning.  The warning breaks DEP-8 tests (because of
-# the stderr output) and is just plain annoying in normal usage.  I don't want
-# to add socks as yet another dependency for pip, nor do I want to allow-stderr
-# in the DEP-8 tests, so just suppress the warning.  pdb tells me this has to
-# be done before the import of pip.vcs.
-from pip._vendor.urllib3.exceptions import DependencyWarning
-warnings.filterwarnings("ignore", category=DependencyWarning)  # noqa
-
-# We want to inject the use of SecureTransport as early as possible so that any
-# references or sessions or what have you are ensured to have it, however we
-# only want to do this in the case that we're running on macOS and the linked
-# OpenSSL is too old to handle TLSv1.2
-try:
-    import ssl
-except ImportError:
-    pass
-else:
-    # Checks for OpenSSL 1.0.1 on MacOS
-    if sys.platform == "darwin" and ssl.OPENSSL_VERSION_NUMBER < 0x1000100f:
-        try:
-            from pip._vendor.urllib3.contrib import securetransport
-        except (ImportError, OSError):
-            pass
-        else:
-            securetransport.inject_into_urllib3()
+# We ignore certain warnings from urllib3, since they are not relevant to pip's
+# usecases.
+from pip._vendor.urllib3.exceptions import (
+    DependencyWarning,
+    InsecureRequestWarning,
+)
 
+import pip._internal.utils.inject_securetransport  # noqa
```

Comment:
I think there should be a code comment here saying why this import is out of order (and so why the error is being ignored).

Reply:
Oh, this is a plain import, not a from...import...

That's why it gets sorted to the top.